shell/app/node_main.cc

```diff
@@ -138,10 +138,16 @@ int NodeMain(int argc, char* argv[]) {
     // Initialize gin::IsolateHolder.
     JavascriptEnvironment gin_env(loop);
 
+    v8::Isolate* isolate = gin_env.isolate();
```

Comment:
the isolate we were pulling out of the Node.js env is the same as the one in the gin env, so this simply makes that clearer to the reader.